src/views/officer-table.j2

```diff
@@ -0,0 +1,29 @@
+<table>
+	<thead>
+		<tr>
+			{% for field in metadata.fields %}
+				<th>{{ field.Label }}</th>
+			{% endfor %}
+		</tr>
+	</thead>
+
+	<tbody>
+		{% for record in records %}
+			{# records are ordered by date, starting with the present into the past, so the "previous" record will be the next item in the list of records #}
+			{% set previous_record = loop.nextitem %}
+			<tr>
+				{% for field in metadata.fields %}
+					{% set classname = record | diff_classname(previous_record, field) %}
+					<td class="{{ classname }}">
```

Comment:
Since we're not using `classname` later in this section, we can probably reduce this to:
```suggestion
					<td class="{{ record | diff_classname(previous_record, field) }}">
```